__tests__/integration.test.js

```diff
@@ -39,65 +39,86 @@ describe("/api/topics", () => {
 });
 
 describe("/api", () => {
-  test.only("should respond with an object describing all endpoints", () => {
+  test("should respond with an object describing all endpoints", () => {
     return request(app)
       .get("/api")
       .expect(200)
       .then((response) => {
-       console.log(response.body, '<-- response body.api ');
-       expect(response.body).toEqual(apiEndpointsJSON);
-    //    console.log(response, '<--- response')
-    //    console.log(response.body.api, '<-- resp.body.api')
-    //    expect(response.body.api).toEqual(apiEndpointsJSON)
+        console.log(response.body, "<-- response body.api ");
+        expect(response.body).toEqual(apiEndpointsJSON);
+        //    console.log(response, '<--- response')
+        //    console.log(response.body.api, '<-- resp.body.api')
+        //    expect(response.body.api).toEqual(apiEndpointsJSON)
       });
   });
 });
 
 describe("/api/articles/:article_id", () => {
   test("GET:200 sends a specific article to the client", () => {
     return request(app)
-      .get('/api/articles/1')
+      .get("/api/articles/1")
       .expect(200)
       .then((response) => {
         expect(response.body.article[0]).toMatchObject({
-            author: expect.any(String),
-            title: expect.any(String),
-            article_id: 1,
-            body: expect.any(String),
-            topic: expect.any(String),
-            created_at: expect.any(String),
-            votes: expect.any(Number),
-            article_img_url: expect.any(String)
-        })
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: 1,
+          body: expect.any(String),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
```

Comment:
At the very least you are getting the article id. However you are also able to know exactly what article 1 looks like. The only one where expect any string is more valid is the created at as it'll change on the reseeding.